test/chi/components/data-table.pug

```diff
@@ -0,0 +1,1347 @@
+---
+title: Data Table
+---
+
+- const sizes = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+.test-base
```

Comment:
```suggestion
.test-base.-w--100
```
Please, provide the class to all the test cases as currently, they render with a narrow width.